packages/samples/headless-ssr/cypress/e2e/smoke.cy.ts

```diff
@@ -14,39 +14,60 @@ const timestampSelector = '#timestamp';
 const resultListSelector = '.result-list li';
 const searchBoxSelector = '.search-box input';
 const routes = ['generic', 'react'] as const;
+const nodeEnv = process.env.NODE_ENV;
+const nextJSRouter = Cypress.env('NEXTJS_ROUTER');
+
 // Note: Thresholds might need to be adjusted as the page tested changes (e.g. more components are added etc)
-const vitals: Record<(typeof routes)[number], Cypress.ReportWebVitalsConfig> = {
-  generic: {
-    thresholds: {
-      // TODO: Remove temporary threshold bump for pages router dev mode
-      // https://coveord.atlassian.net/browse/KIT-2834
-      // fcp: 200,
-      // lcp: 200,
-      fcp: 2000,
-      lcp: 2000,
-      cls: 0,
-      ttfb: 60,
-      // TODO: Ensure validity of following input based vitals with interactive elements
-      fid: 400,
-      inp: 400,
-    },
-  },
-  react: {
-    thresholds: {
-      // TODO: Remove temporary threshold bump for pages router dev mode
-      // https://coveord.atlassian.net/browse/KIT-2834
-      // fcp: 400,
-      // lcp: 400,
-      fcp: 2000,
-      lcp: 2000,
-      cls: 0,
-      ttfb: 120,
-      // TODO: Ensure validity of following input based vitals with interactive elements
-      fid: 800,
-      inp: 800,
-    },
-  },
-};
+const vitals: Record<(typeof routes)[number], Cypress.ReportWebVitalsConfig> =
+  nodeEnv === 'development' && nextJSRouter === 'pages'
```

Comment:
From the test failures (in CI) looks like this is not working as expected?

Reply:
yes looks like there is a problem,. Im pretty sure it was working locally !

Reply:
Wonder if process.env is not working as expected in CI or if next js or something else is setting NODE_ENV to a different value based on the CI env.

Reply:
Just a silly mistake on my part forgetting to put it in e2e instead of e2e:watch !